nntrainer/graph/network_graph.h

```diff
@@ -146,6 +146,24 @@ class NetworkGraph {
    */
   void setBatchSize(unsigned int batch_size);
 
+  /**
+   * @brief try zero gradient at the start of gradient access
+   * @note if it is not the start of gradient access, this is noop
+   *
+   * @param node node to try gradient to zero
+   */
+  void tryZeroGrad(LayerNode *node);
```

Comment:
I wonder why this is `try`?

Reply:
`try` makes it seems like this can error, but this will always succeed. This is more like it will make gradient zero where it is required. Recommend you to remove `try`.

Reply:
I'll rename it to zeroGradOnFirstAccess

just zeroGrad will look like this code is buggy at seeing the usage at the very first.

Reply:
sounds good.

Reply:
Done